pyomo/gdp/plugins/bigm.py

```diff
@@ -208,23 +206,12 @@ def _apply_to_impl(self, instance, **kwds):
         # in the tree rooted at instance.
         knownBlocks = set()
         for t in targets:
-            # [ESJ 08/22/2019] This can go away when we deprecate CUIDs. The
-            # warning is in util, but we have to deal with the consequences here
-            # because we need to have the instance in order to get the
-            # component.
-            if isinstance(t, ComponentUID):
-                tmp = t
-                t = t.find_component(instance)
-                if t is None:
-                    raise GDP_Error(
-                        "Target %s is not a component on the instance!" % tmp)
-
             # check that t is in fact a child of instance
-            if not is_child_of(parent=instance, child=t,
+            elif not is_child_of(parent=instance, child=t,
```

Comment:
```suggestion
            if not is_child_of(parent=instance, child=t,
```

Otherwise you're missing an initial `if`.